nc/source/ecs/component/Component.cpp

```diff
@@ -5,9 +5,9 @@
 
 namespace nc
 {
-    void AutoComponent::ComponentGuiElement()
+    void FreeComponent::ComponentGuiElement()
     {
-        ImGui::Text("User AutoComponent");
+        ImGui::Text("Unknown FreeComponent");
```

Comment:
When would this be hit?

Reply:
For any user-defined FreeComponent that doesn't override it. It still shows up in the editor as a component, but we don't know how to present more data than "something exists here."